Assets/Scripts/OrchestraArmy/Room/RoomController.cs

```diff
@@ -1,433 +1,61 @@
 using System;
 using System.Collections.Generic;
-using UnityEngine;
+using OrchestraArmy.Entity.Entities.Enemies;
 using OrchestraArmy.Event;
 using OrchestraArmy.Event.Events.DoorAccess;
+using OrchestraArmy.Event.Events.Enemy;
 using OrchestraArmy.Event.Events.Player;
+using UnityEngine;
+using Object = UnityEngine.Object;
 using UnityEngine.Experimental.Rendering;
 using Random = UnityEngine.Random;
+using UnityEditor.AI;
```

Comment:
Zijn de .AI imports nodig? worden niet gebruikt zover ik zie en ze breken de build

Reply:
Is van Bram